packages/node/src/utils/meta.ts

```diff
@@ -13,25 +13,23 @@ import {
 } from '@sentry/utils';
 
 /**
- * Extracts the tracing data from the current span or from the client's scope
- * (via transaction or propagation context) and renders the data to <meta> tags.
+ * Extracts the tracing data from the current span or from the client's scope (via transaction or propagation context)
+ * and serializes the data to <meta> tag contents.
  *
- * This function creates two serialized <meta> tags:
- * - `<meta name="sentry-trace" content="..."/>`
- * - `<meta name="baggage" content="..."/>`
- *
- * TODO: Extract this later on and export it from the Core or Node SDK
+ * Use this function to obtain the tracing meta tags you can inject when rendering an HTML response to continue
+ * the server-initiated trace on the client.
  *
  * @param span the currently active span
  * @param client the SDK's client
  *
- * @returns an object with the two serialized <meta> tags
+ * @returns an object with the two meta tags. The object keys are the name of the meta tag,
+ * the respective value is the content.
  */
 export function getTracingMetaTags(
```

Comment:
m: Can we think of a different name here? IMHO this sounds to me as if that would return the actual `tag` elements, but it returns strings. What about e.g. `getTraceData` or something along these lines...?

Reply:
That's a good point. I chose this return type because returning one (or two) string(s) with rendered HTML is too specific of a return type to cover all use cases (e.g. see https://github.com/getsentry/sentry-javascript/issues/8843#issuecomment-2040038590). 
I couldn't really come up with a better return type that's still versatile enough but simpler than the current one to use.

So I guess we give it a more general name then. Just a bit unhappy that it's not as simple as in [laravel](https://docs.sentry.io/platforms/php/guides/laravel/tracing/distributed-tracing/).

Reply:
I think the return type is good (more generic, and easier to port)! We can also use something like `getTraceMetaTagValues`, for example? 🤔